src/vmm/src/mmds/data_store.rs

```diff
@@ -65,6 +70,7 @@ impl Default for Mmds {
 }
 
 impl Mmds {
+    /// MMDS default instance with limit {data_store_limit:?}
```

Comment:
The standard for referring to variables and such is using markdown codeblocks. Our error enums use the rust-formatter syntax `{...:?}` instead because they derive display doc, which causes the interpolation to happen at runtime if the error is printed somewhere. However, for non-error docstrings, no interpolation will never happen

```suggestion
    /// MMDS default instance with limit `data_store_limit`
```